functests/4_latency/latency.go

```diff
@@ -99,7 +99,19 @@ var _ = Describe("[performance] Latency Test", func() {
 		Expect(err).ToNot(HaveOccurred(), "error looking for the optional selector: %v", err)
 
 		Expect(workerRTNodes).ToNot(BeEmpty())
-		workerRTNode = &workerRTNodes[0]
+
+		//At least one worker node should have cpu.Allocatable greater than the quantity requested by each test, else skip the test
```

Comment:
This is a good change. It should be moved on a new function in `functests/utils/nodes/nodes.go` (or like that)